domain/schema/model_schema_test.go

```diff
@@ -247,6 +247,7 @@ func (s *modelSchemaSuite) TestModelTables(c *tc.C) {
 		"storage_volume_status",
 		"storage_volume_status_value",
 		"unit_storage_directive",
+		"storage_provision_scope",
```

Comment:
An effort has been made here to keep this list alphabetically ordered. Can we move this up to keep the consistency?